Resources/Prototypes/_DV/Entities/Objects/Misc/land_mine.yml

```diff
@@ -0,0 +1,27 @@
+- type: entity
+  name: explosive mine
+  suffix: DeltaV
+  parent: BaseLandMine
+  id: LandMineHE
```

Comment:
parent -> id -> name -> suffix
the suffix should also be changed to High Explosive so its less opaque as to what it is